plugins/premium/tinymcespellchecker.md

```diff
@@ -204,19 +204,32 @@ tinymce.init({
 });
 ```
 
-### `spellchecker_whitelist`
+### `spellchecker_ignorelist`
 
 This option specifies an array of words to be ignored by the spell checker.
```

Comment:
Do we need to update this to say something like this maybe?

```suggestion
This option specifies which words should be ignored by the spell checker. If an array of words is provided, the words will be ignored for all languages. If an object providing an array of words per language is provided, the words will be ignored for the specified languages.
```

Reply:
```suggestion
This option specifies which words should be ignored by the spell checker. If an array of words is provided, the specified words will be ignored for all languages. If an object containing an array of words per language is provided, the specified words will be ignored for the specified languages.
```
maybe? had `provided` a lot, and I got more specific.